src/app/core/ui/ui/ui.component.html

```diff
@@ -42,7 +42,7 @@
     class="flex-row align-center flex-grow-1-3 justify-content-end gap-small"
   >
     @if (isLoggedIn()) {
-      <app-search class="hide-mobile full-width"></app-search>
+      <app-search [ngClass]="{ 'full-width': isDeskotp }"></app-search>
```

Comment:
```suggestion
      <app-search [ngClass]="{ 'full-width': isDesktop }"></app-search>
```

or could use the `[ngClass.full-width]=""` style also